ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -245,35 +239,41 @@ protected void onWindowVisibilityChanged(int visibility) {
                     context.getApplicationContext(), mAdapterId, new VungleInitializer.VungleInitializationListener() {
                         @Override
                         public void onInitializeSuccess() {
-                            loadAdForBanner();
+                            loadBanner();
                         }
 
                         @Override
                         public void onInitializeError(String errorMessage) {
-                            Log.w(VungleMediationAdapter.TAG, "Failed to load ad from Vungle: " + errorMessage);
+                            Log.w(TAG, "Failed to load ad from Vungle: " + errorMessage);
                             if (mMediationBannerListener != null) {
                                 mMediationBannerListener.onAdFailedToLoad(VungleInterstitialAdapter.this,
                                         AdRequest.ERROR_CODE_INTERNAL_ERROR);
                             }
                         }
                     });
         } else {
-            Log.w(VungleMediationAdapter.TAG,
-                    "Failed to load ad from Vungle: Missing or Invalid Placement ID");
+            StringBuilder errMsg = new StringBuilder("Failed to load ad from Vungle: ");
+            if(!isPlacementValid) {
+                errMsg.append("Missing or Invalid Placement ID. ");
+            }
+            if (!isBannerSizeValid) {
+                errMsg.append("Invalid banner size. ");
+            }
+            Log.w(TAG, errMsg.toString());
             mMediationBannerListener
                     .onAdFailedToLoad(VungleInterstitialAdapter.this,
                             AdRequest.ERROR_CODE_INVALID_REQUEST);
         }
     }
 
-    private void loadAdForBanner() {
+    private void loadBanner() {

```

Comment:
hopefully we dont confuse AdMob and ourselves, as other ad networks use load term to show/render, where we use it for caching